.github/workflows/draw-zmk.yml

```
@@ -44,6 +44,11 @@ on:
         default: false
         required: false
         type: boolean
+      install_repo:
+        description: 'Install keymap-drawer from a git repo. Primarily for testing changes using a keymap-drawer fork.'
+        default: 'caksoylar/keymap-drawer'
+        required: false
+        type: string
```

```suggestion
      install_repo:
        description: 'Install keymap-drawer from a different git remote, primarily for testing changes using a keymap-drawer fork. Ignored if `install_branch' is unset/empty.
        default: 'https://github.com/caksoylar/keymap-drawer.git'
        required: false
        type: string
```

Tweaked the description a bit and made it the full remote URI, so you can use other git forges.

---

Can you also move this to below `install_branch`, since this depends on the former?